pandas/conftest.py

```diff
@@ -1,14 +1,14 @@
 import importlib
 import os
 
-import hypothesis
-from hypothesis import strategies as st
 import numpy as np
 import pytest
 
 from pandas.compat import PY3
 import pandas.util._test_decorators as td
 
+import hypothesis
+from hypothesis import strategies as st
```

Comment:
Not very important, but I don't understand this change. The standard is to import Python stdlib first, third-party packages (like hypothesis) later, and the project code (pandas) last.

Reply:
isort did that, I ran it on the whole repository

Reply:
So whenever somebody else runs it and since pandas/conftest.py is not in setup.cfg, they should get this change also with isort. Unless something went wrong.

Reply:
That's weird. Not so important as I said, but if you have time, I'd remove this file from the PR, as nothing really changed here more than the isort changes, which are unrelated. Or if you want to keep, may be we should remove this file from the excludes of isort (assuming it's there).

Reply:
@alimcmaster1 can you take a look here? May be I'm missing something?

Reply:
It's already been removed from the excludes of isort in setup.cfg. That's why isort picked it up and changed it. Should I manually change it back and remove it from this pr? Because the next person that runs isort should get this change also.

Reply:
Looks like the CI agrees with me:
```
ERROR: /home/travis/build/pandas-dev/pandas/pandas/conftest.py Imports are incorrectly sorted.
ERROR: /home/travis/build/pandas-dev/pandas/pandas/tests/io/test_sql.py Imports are incorrectly sorted.
```

It's probably a bug in `isort` when sorting the files, validation seems right.

Can you fix, so the CI is green and we can merge?

Reply:
Thanks @datapythonista will take a look at what is going on with isort and that file this evening.

Reply:
I reverted everything that isort did and added test_sql.py back to setup.cfg. I tried running isort a few times, but it just sorts everything the way it sorted the first time.

Reply:
Running `isort conftest.py` ran OK for me on master. It should already be sorted fine as its not in the setup.cfg.

Reply:
@alimcmaster1 My version is 4.3.4 as well.